include/voltorb_flip/voltorb_flip_game.h

```diff
@@ -105,26 +105,26 @@ typedef struct BoardConfig {
     u8 maxFreeMultipliers : 4;
 } BoardConfig;
 
-GameState *CreateGameState(enum HeapID);
-void FreeGameState(GameState *);
-void NewBoard(GameState *);
-void ov122_021E8528(GameState *);
-void SetRoundOutcome(GameState *, RoundOutcome);
-void MultiplyPayoutAndUpdateCardsFlipped(GameState *, CardType);
-void FlipCard(GameState *, int);
-BOOL DeductFromPayout(GameState *, u8);
-BOOL IsCardFlipped(GameState *, int);
-BOOL EarnedMaxPayout(GameState *);
-CardType GetCardType(GameState *, int);
-int IsCardMemoFlagOn(GameState *, int, int);
-void ToggleCardMemo(GameState *, int, int);
-int PointsAlongAxis(GameState *, Axis, u8);
-int VoltorbsAlongAxis(GameState *, Axis, u8);
-int FlippedCardsAlongAxis(GameState *, Axis, u8);
-u16 GamePayout(GameState *);
-u8 MultiplierCards(GameState *);
-u8 MultiplierCardsFlipped(GameState *);
-u8 GameLevel(GameState *);
-int LevelsGained(GameState *);
+VoltorbFlipGameState *VoltorbFlip_CreateGameState(enum HeapID);
+void VoltorbFlip_FreeGameState(VoltorbFlipGameState *);
+void VoltorbFlipGameState_NewBoard(VoltorbFlipGameState *);
```

Comment:
Consistency/Non-Blocking: We usually use _New when creating a pointer to some struct, but contextually it also makes sense here. Might be worth changing tho for lack of confusion.

Suggestion: `_Init(ialize)Board`